binaries/scheduler/main.go

```diff
@@ -17,18 +17,14 @@ import (
 var addr = flag.String("addr", "localhost:9090", "Bind address for api server.")
 var httpPort = flag.Int("http_port", 9091, "port to serve http on")
 
-var configFileName = flag.String("config", "local.json", "Scheduler Config File")
+var configFlag = flag.String("config", "local.inmemory", "Scheduler Config (either a filename like local.inmemory or JSON text")
 
 func main() {
-
 	flag.Parse()
-	log.Printf("Scheduler: reading config %+v", *configFileName)
-	config, err := config.Asset(fmt.Sprintf("config/%v", *configFileName))
-
+	configText, err := server.GetConfigText(*configFlag, config.Asset)
 	if err != nil {
-		log.Fatalf("Error Loading Config File: %v, with Error: %v", configFileName, err)
+		log.Fatal(err)
```

Comment:
This error will have that context, so this change doesn't lose any info.

It's populated by scootapi/server/setup.go:GetConfigText, on line 109.